pipeline/00-ingest.R

```diff
@@ -49,9 +50,10 @@ training_data <- dbGetQuery(
   INNER JOIN default.vw_pin_sale sale
       ON sale.pin = res.meta_pin
       AND sale.year = res.year
-  WHERE res.year
-      BETWEEN '{params$input$min_sale_year}'
-      AND '{params$input$max_sale_year}'
+  WHERE CAST(res.year AS int)
```

Comment:
Need to slightly expand the training data pull here to capture sales earlier than 2015 for the count feature.